packages/text-bitmap/src/BitmapFont.ts

```diff
@@ -18,15 +21,18 @@ interface IBitmapFontCharacter {
 // Internal map of available fonts, by name
 const available: Dict<BitmapFont> = {};
 
+// Default dimensions of the bitmap font atlas texture
+const BMT_DIMEN = 1024;
+
+// Default padding b/w glyphs in the bitmap font atlas
+const BMT_PADDING = 4;
```

Comment:
It would be nice if these settings weren't hard coded in here, but could be adjusted by developers. Maybe they're part of the settings passed in via 'from'? So can be adjusted on a case by case basis.

Reply:
Good call